### ISSUE TITLE
Error while saving

### ISSUE THREAD

**Issue by GuthrieDyce** [csn-le/wave_clus#188]
Error while trying to save clusters
Can not write file
[filepath].

Error in wave_clus>save_clusters_button_Callback (line 547)
eval(['save( ''' outfile ''',''' var_list '' ver ');']);

Error in gui_mainfcn (line 95)
        feval(varargin{:});

Error in wave_clus (line 63)
    gui_mainfcn(gui_State, varargin{:});
 
Error while evaluating UIControl Callback

I am using matlab 2016b on windows 10

Please help

**Comment by ferchaure** [comment]
Hi, did you try in another folder? remember to download the latest version of the code [here](https://github.com/csn-le/wave_clus/archive/master.zip)

**Comment by GuthrieDyce** [comment]
Hi there,
I did try saving to another folder but it did not work.  
It also occurred to me that I was saving onto a network drive, so I tried saving to my desktop, but that also did not work.
I will try updating the code and let you know if it fixes the problem.

**Comment by GuthrieDyce** [comment]
Having replaced my version with a new version the error is fixed.
I noticed that I had copied everything I thought I needed from a master folder to a sub-directory.
If Matlab was running wave_clus from that other directory and not all of the dependencies were copied to that directory the error might stem from that and not a bug in an old version of wave_clus.
Having said that it is strange that the error did not always occur (sometimes data saves properly).
At any rate, I'm leaving everything in the master folder this time.
I won't close this thread just yet in case the error does crop up again for me or someone else.
